packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-point.test.tsx

```diff
@@ -48,66 +67,51 @@ describe("MovablePoint", () => {
         labels: [],
     };
 
+    const graphConfigContextWithTooltips = {
+        ...baseGraphConfigContext,
+        showTooltips: true,
+    };
+
     let userEvent: UserEvent;
     beforeEach(() => {
-        useGraphConfigMock = jest.spyOn(ReducerGraphConfig, "default");
-        useDraggableMock = jest
-            .spyOn(UseDraggableModule, "useDraggable")
-            .mockReturnValue({dragging: false});
         userEvent = userEventLib.setup({
             advanceTimers: jest.advanceTimersByTime,
         });
     });
 
     describe("Tooltip", () => {
-        const graphConfigContextWithTooltips = {
-            ...baseGraphConfigContext,
-            showTooltips: true,
-        };
-
-        /*  NOTE: When a WonderBlocks Tooltip is added to an element, it isn't rendered with the element.
-                  Instead, it is rendered elsewhere in the document (usually the bottom of the <body>),
-                        and only when the user interacts with the element.
-                  This makes checking for the Tooltip element difficult to manage in a test.
-                  Therefore, these tests mock the Tooltip component and check if/how it is called.
-         */
-        beforeEach(() => {
-            // @ts-expect-error // TS2339: Property mockImplementation does not exist on type typeof Tooltip
-            Tooltip.mockImplementation(TooltipMock);
```

Comment:
Mocking the tooltip in `beforeEach()` can cause problems when tests are run in parallel. Plus, all we really want to do is verify that the content of the tooltip was rendered and so we can mock the module for all tests (see higher up in this file).